### DISCUSSION TITLE
Dealing with corrupted pdfs

### DISCUSSION THREAD

**Discussion by rajban94** [jsvine/pdfplumber#690] [Q&A]
In the try block i have wrote with pdfplumber.open(pdf_file) as pdf but as the pdf file is corrupted it is throwing error in the except block now how can i close that pdf in order to move that pdf to another folder because whenever i tried using os module inside the except block it is showing me permission error that the pdf is used in another process just because the pdf is not closed. If you can give me an idea or code block that will help me to get rid of this problem.

**Comment by samkit-jain** [comment]
Hi @rajban94 Appreciate your interest in the library. The open function does indeed closes the file as you can see at https://github.com/jsvine/pdfplumber/blob/f9c1f6172b47a24cd4356f20711a52d6850397fc/pdfplumber/pdf.py#L89 Could you please share the PDF, and some reproducible code?
